code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/data/LoadBalancerTargetsInfo.java

```diff
@@ -11,10 +11,22 @@
     List<LoadBalancerTargetInfo> targets;
     InstanceHealthCheck healthCheck;
 
-    public LoadBalancerTargetsInfo(List<LoadBalancerTargetInfo> lbTargetInfo, InstanceHealthCheck healthCheck) {
+    public LoadBalancerTargetsInfo(List<LoadBalancerTargetInfo> lbTargetInfo, InstanceHealthCheck lbHealthCheck) {
         super();
         this.targets = lbTargetInfo;
-        this.healthCheck = healthCheck;
+        for (LoadBalancerTargetInfo target : this.targets) {
+            if (target.getHealthCheck() != null) {
```

Comment:
Yes, as on that level where we ignore the rest, we read only one parameter from the healtcheck - requestLine. On the target level, each target will get its own healthcheck configured anyway